atlas_installer/build.sh

```diff
@@ -1,15 +1,16 @@
 #!/bin/bash
 
 export build_version=$(python get_version.py)
+release_version=$(echo $build_version | sed "s/.dev[0-9]*//")
```

Comment:
I think the change is great ... I like the option to use regex to get rid of the dev from the version ... I think @amackillop you should also consider that there are several things that use this version strategy ... I'm wondering what else this will impact....

Reply:
This particular change only seems to impact which URL is used in the installer to download the release from github. I left the exporting of the original build version in case that is pulled from somewhere later but I don't think that it is.